crates/store/re_video/src/decode/mod.rs

```diff
@@ -55,5 +60,6 @@ pub struct Frame {
 }
 
 pub enum PixelFormat {
+    Rgb8Unorm,
```

Comment:
This is the FFMPEG output

Reply:
it doesn't have to be though, we can choose that fairly freely, no?

Reply:
Not completely freely. We can pcik `rgb24`, but not `rgba32`, for instance

Reply:
We can pick some yuv formats for sure, but we didn't have GPU support for that when I wrote this code.